Octokit.Tests.Integration/Helpers/DisposableRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit.Tests.Integration.Helpers
+{
+    public class DisposableRepository : Repository, IDisposable
+    {
+        public static DisposableRepository InitFromRepository(Repository repository)
```

Comment:
Rename this to `Initialize` - I think there's enough detail from the API surface to indicate we're working with repositories here.